src/python/install.sh

```diff
@@ -396,23 +454,23 @@ install_from_source() {
             ;;
     esac
 
-    # Download tgz of source
-    mkdir -p /tmp/python-src ${INSTALL_PATH}
-    cd /tmp/python-src
-    local tgz_filename="Python-${VERSION}.tgz"
-    local tgz_url="https://www.python.org/ftp/python/${VERSION}/${tgz_filename}"
-    echo "Downloading ${tgz_filename}..."
-    curl -sSL -o "/tmp/python-src/${tgz_filename}" "${tgz_url}"
-
+    install_cpython "${VERSION}"
+    if [ -f "/tmp/python-src/${cpython_tgz_filename}" ]; then 
+        if grep -q "404 Not Found" "/tmp/python-src/${cpython_tgz_filename}"; then
+            # Use grep to search for "404 Not Found" in the file
+            echo "\"404 Not Found\" found in /tmp/python-src/${cpython_tgz_filename}. Not able to create source binary"
```

Comment:
Looks like this was used for debugging, remove?

Reply:
removed 👍